cc/mcts_player.h

```diff
@@ -92,6 +92,23 @@ class MctsPlayer {
     // move is played.
     bool tree_reuse = true;
 
+    // "Playout Cap Oscillation" as per the KataGo paper.
+    // If fastplay_frequency > 0, tree search is modified as follows:
```

Comment:
does it make sense to express this in terms of 'slowplay frequency'? I find numbers like 0.02 more semantically meaningful than 0.98, but it's a very asethetic concern and consistency with the katago paper is probably more important.

Reply:
in that case i should probably reverse these, because katago specifies the slowplay % XD

Reply:
actually thinking about it further, all the existing doc assumes "slowplay" mode only.  Rather than rename everything, the 'fastplay' is the new behavior.